src/utils/Errors.ts

```diff
@@ -1,5 +1,6 @@
 import { error } from "vscode-extension-tester";
 
+// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
I am not sure if I can change this to match requirement because I don't correctly understand the background of this file. Can you take a peak please @djelinek?

Reply:
we can skip that for now